src/jira-client.js

```diff
@@ -1,89 +1,114 @@
+const { format, URL, URLSearchParams } = require('url');
 const rp = require('request-promise');
+const diagnostics = require('diagnostics');
 const { getCurrentContext } = require('./config');
 
+const debug = {
+  http: diagnostics('jiractl:http'),
+  verbose: diagnostics('jiractl:verbose')
+};
+
 let sessionCookie;
+const authmode = 'basic';
```

Comment:
I would be okay merging with this hard-coded but we really need to make sure that we can set it via a new CLI command before we publish.